scrapers/victrola.py

```diff
@@ -58,11 +59,12 @@ def scrape_victrola():
             except:
                 notes = coffee_soup.find(text="Tasting Notes").next_element.strip()[2:].rstrip(',').lower().split(',')
                 pass
-            try:
-                region = coffee_soup(text=re.compile(r'Region:'))[1][8:]
-            except:
-                region = 'n/a'
-                pass
+            is_country_in_here = [x for x in countrydict.keys() if x in name.lower()]
+            if len(is_country_in_here) != 0:
```

Comment:
in python, empty list counts as a falsey value, so you can just do `if not is_country_in_here:`